Cargo.toml

```diff
@@ -41,24 +41,23 @@ travis-ci = { repository = "tokio-rs/tokio" }
 appveyor = { repository = "carllerche/tokio", id = "s83yxhy9qeb58va7" }
 
 [dependencies]
-tokio-current-thread = { version = "0.1.0", path = "tokio-current-thread" }
```

Comment:
I'm pretty sure that the version numbers are needed here in order to push the crates to crates.io. I'd recommend reverting this block.

Reply:
> However, crates that use dependencies specified with only a path are not permitted on crates.io.

Yes you are right